lib/snapshot.js

```diff
@@ -182,7 +186,15 @@ function take_snapshot( p_uri, p_filename, p_width, p_height, screen_width, scre
 			});
 
 		if ( response && response.ok() || page_loaded ) {
-			await page.waitFor( 2000 );
+			try {
```

Comment:
Just cosmetically It'd be nice to match the formatting of the error handling in `page.goto()` and put the error handling up with the promise, like
```
page.waitForNavigation(…).
    catch( e => { if ( error instanceof _puppeteer.errors.TimeoutError ) … } ) 
  ```
```

Reply:
Updated this :)